niksmr/vue-masked-input
# Browser compatibility fixes

This PR contains various browser compatibility fixes.

Browsers tested:
* Latest Chrome on Linux
* Latest Chrome on Android
* Chrome 30 on Android
* Latest Firefox on Linux
* Latest Edge on Windows
* IE 11 on Windows
* IE 11 on Windows Phone
* Latest Safari on iPhone

Features tested:
* Inserting only letters
* Inserting only numbers
* Inserting only letters on part of field, inserting only number on other part of same field
* Having a prefix
* Backspace
* Set value using .value with vanilla js
* Paste (for IE11 see https://github.com/niksmr/vue-masked-input/pull/45)
